gson2/src/test/java/org/jdbi/v3/gson2/TestGson2Plugin.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.gson2;
+
+import java.util.List;
+import org.assertj.core.groups.Tuple;
+import org.jdbi.v3.core.Handle;
+import org.jdbi.v3.json.Json;
+import org.jdbi.v3.postgres.PostgresDbRule;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+import org.jdbi.v3.sqlobject.statement.SqlUpdate;
+import org.jdbi.v3.testing.JdbiRule;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestGson2Plugin {
```

Comment:
Should this test now be an abstract test superclass in `jdbi3-json` and have two implementing tests, one in `jackson` and one in `gson`?

Reply:
Hmm, I guess it should?

Reply:
I think that's preferable to copy+pasting the code...